collect_app/src/main/AndroidManifest.xml

```diff
@@ -192,7 +192,6 @@ the specific language governing permissions and limitations under the License.
 
                 <category android:name="android.intent.category.DEFAULT" />
 
-                <data android:mimeType="vnd.android.cursor.item/vnd.odk.form" />
```

Comment:
Does this result a breaking change in what external clients can do?

Reply:
I don't think so. It just moves the target to `FormUriActivity`. Both worked with the `ViewFormActionTest`.

Reply:
It will definitely work with the actions. My question is whether integrations could previously have been using the class name. If so, this is breaking.

Reply:
We’ve talked about this and decided to leave as-is. We now document the actions. The class names were never documented and we want to move more in the direction of them not being considered part of the API. 

For folks who upgrade to this and already have an integration directly with the activity, it will leave things in a bad state of the project isn’t first switched. They’ll get no indication of this.